src/dataDisplay/personPanel/personPanelDetails.jsx

```diff
@@ -6,23 +6,24 @@ import {
 } from 'lodash';
 import ClassNames from 'classnames';
 import moment from 'moment-timezone';
+import MomentPropTypes from 'react-moment-proptypes';
```

Comment:
We finally ended up doing it, eh?! :-)
Nice!

Will keep reading, hopefully we also made use of this in good ol' `<DurationPicker />`.

Reply:
I did this in eebfe26.  Resolving.